hiera.yaml

```
@@ -0,0 +1,10 @@
+---
+version: 5
+
+defaults:  # Used for any hierarchy level that omits these keys.
+  datadir: data         # This path is relative to hiera.yaml's directory.
+  data_hash: yaml_data  # Use the built-in YAML backend.
+
+hierarchy:
+  - name: 'common'
+    path: 'common.yaml'
```

```suggestion
    path: 'common.yaml'
```

Needs a newline at the end of the file.